src/Generating/ComposableGenerator.cpp

```diff
@@ -155,27 +155,33 @@ void cComposableGenerator::GenerateBiomes(cChunkCoords a_ChunkCoords, cChunkDef:
 
 void cComposableGenerator::Generate(cChunkDesc & a_ChunkDesc)
 {
+	// Wrap a cChunkDesc::Shape into a structure so that we can make a unique_ptr out of it:
+	struct ShapeHelper
+	{
+		cChunkDesc::Shape mShape;
```

Comment:
xoft pls

just call it `Shape`

Reply:
Or not, it's fine...

Reply:
One of these days Imma write a script that sends all those underscores into oblivion...